test/UrlHelperTest.php

```diff
@@ -48,7 +48,7 @@ public function testRaisesExceptionOnInvocationIfNoRouteProvidedAndResultIndicat
 
     public function testRaisesExceptionOnInvocationIfRouterCannotGenerateUriForRouteProvided()
     {
-        $this->router->generateUri('foo', [])->willThrow(RouterException::class);
+        $this->router->generateUri('foo', [], [])->willThrow(RouterException::class);
```

Comment:
Correct. It's probably better to leave them as-is, actually, so they'll fail if the parameter suddenly becomes required for whatever reason.

I'll address that (and whatever else comes up) when I update / prepare the PR.